### PR TITLE
Update MyBiggerBuffs.lua

### PR DESCRIPTION
**Pull request by KyleQianliMa**
This might be a fix to the actionblocked taint issue. But need more testing to see if it creates other issues. Got this idea from another addon I use.

### CODE REVIEW

**Comment by kpontussvensson** [comment]
If this works then you should also un-comment the things in Gui.lua

File: MyBiggerBuffs.lua [hunk 1]
```diff
@@ -77,30 +81,57 @@ local function createBuffFrames(frame)
...
+  hooksecurefunc('CompactUnitFrame_UpdateAll', function(frame)
+		if frame:IsForbidden() then
+			return
+		end
+		local name = frame:GetName()
+		if not name or not name:match('^Compact') then
+			return
+		end
+		if InCombatLockdown() then
+			if not UpdateTable[frame] then
```
Comment (kpontussvensson) [inline MyBiggerBuffs.lua]: This throws a nil error - the table need to be declared. And you also need to actually do something with the table for it to be useful. It seems to be taken from BigDebuffs where they use it to call the createBuffFrames(frame) on PLAYER_REGEN_ENABLED (= when you get out of combat) which is missing here.

Comment (KyleQianliMa) [reply MyBiggerBuffs.lua]: Good call. I got rid of the table thing. I'm not sure where the original idea comes from. I saw this from an addon Sortgroup first.

**Comment by wkrueger** [comment]
was not checking github notifications...

did this improve the taint issues?

**Pull request closed without merging**